.github/workflows/semgrep_coverage_report.yml

```diff
@@ -60,80 +91,70 @@ jobs:
         while True:
             url = f'https://api.github.com/orgs/{org_name}/repos?page={page}&per_page=100'
             response = requests.get(url, headers=headers)
-            logging.debug("Requesting list of repos in Org: %s, page: %s" ,org_name, page)
-            logging.debug("Response Code: %s", response.status_code)
+            logging.debug(f"Requesting list of repos in Org: {org_name}, page: {page}")
+            logging.debug(f"Response Code: {response.status_code}")
             page_repos = response.json()
             if not page_repos:
                 break
             repos.extend(page_repos)
             page += 1
-        
-        # response = requests.get(f'https://api.github.com/orgs/{org_name}/repos', headers=headers)
-        # repos = response.json()
 
-        # Read the CSV files
         daily_df = pd.read_csv('daily.csv', header=None)
         weekly_df = pd.read_csv('weekly.csv', header=None)
         
-        # Convert to lists for easier checking
         daily_repos = daily_df[0].tolist()
         weekly_repos = weekly_df[0].tolist()
         github_repos = [repo['name'] for repo in repos]
 
         daily_repos_not_in_github_repos = [item for item in daily_repos if item not in github_repos]
         weekly_repos_not_in_github_repos = [item for item in weekly_repos if item not in github_repos]
 
-        # Convert list to DataFrame
         df_daily_repos_not_in_github_repos = pd.DataFrame(daily_repos_not_in_github_repos)
         df_weekly_repos_not_in_github_repos = pd.DataFrame(weekly_repos_not_in_github_repos)
-        
-        # Write DataFrame to CSV
+
         df_daily_repos_not_in_github_repos.to_csv('daily_repos_not_in_github_repos.csv', index=False)
         df_weekly_repos_not_in_github_repos.to_csv('weekly_repos_not_in_github_repos.csv', index=False)
 
```

Comment:
add debug "daily_repos_not_in_github_repos written to file = "
add debug "weekly_repos_not_in_github_repos written to file"